### DISCUSSION TITLE
Repetitive Alert notifications question

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#99] [Q&A]
Running the new codebase is great..  I do have a request on the SMS and Discord notifications. When an alert hits, it keeps alerting via SMS and Discord multiple times until the product is OOS. Is there a way to set the bot to send repetitive alerts every X minutes or none at all? Just alert once? 

![image](https://user-images.githubusercontent.com/74942520/116898297-32f9a400-ac04-11eb-9657-a08a201031ba.png)

**Comment by Prince25** [comment]
It is designed to only send alert once for an item per session as long as it doesn't go OOS and then restock. As you can see, there are 3 instances in that screenshot where XBOX Series X is in stock at Costco and it doesn't send a notification. The only thing that would cause it to send notification again is if it thought the item was OOS for some reason. I think this happens at `11:27:17 am` because it didn't say it was still in stock. Does it happen with a particular item or store, or does it happen in every store?

**Comment by xmpierce1** [comment]
Good observation. Honestly, I do not know, but Costco has been known to flicker on the OOS with the consoles.

**Comment by Prince25** [comment]
That's most likely why it's happening. Hmm, Not sure what's the best way to handle this then.
